config/env/test.js

```diff
@@ -28,6 +28,14 @@ module.exports = {
   app: {
     title: defaultEnvConfig.app.title + ' - Test Environment'
   },
+  uploads: {
+    profileUpload: {
```

Comment:
Good point. I see that in Trustroots you simplified function names like profileUploadFileFilter too, because that function really works on any image, not just the profile uploader.